src/main/java/nl/tudelft/ti2206/group9/gui/AbstractPopup.java

```diff
@@ -0,0 +1,88 @@
+package nl.tudelft.ti2206.group9.gui;
+
+import java.util.Arrays;
+
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Button;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Rectangle;
+import javafx.stage.Popup;
+
+/**
+ * Abstract Popup class as a template for all Popups in the game.
+ * @author Maarten, Robin
+ */
+@SuppressWarnings("restriction")
+public abstract class AbstractPopup extends Popup {
+
+	/** Width of the PopUpMenu. */
+	public static final double WIDTH = 260;
+	/** Height of the PopUpMenu. */
+	public static final double HEIGHT = 320;
+	/** Size of the HBox. */
+	protected static final double HBOX_SIZE = 10;
+	/** Size of the VBox. */
+	protected static final double VBOX_SIZE = 50;
```

Comment:
Technically this is not the size of the box, rather the spacing between the elements.

< naming